packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -1407,9 +1399,9 @@ class InspectorInstanceRef {
   int get hashCode => id.hashCode;
 
   @override
-  String toString() => id;
+  String toString() => id!;
```

Comment:
this will fail if id is null. either make id non-nullable or give the toString method a default value 'null' if id is null

Reply:
done